### DISCUSSION TITLE
Autocompletion in python

### DISCUSSION THREAD

**Discussion by VedAustin** [zed-industries/zed#6518] [Feature Requests]
Hi .. excited to try this new IDE. As a python user, I am unable to get autocompletion to work. Am I missing a setup step?
```
(base) ➜  ~ node -v
v14.17.0
(base) ➜  ~ npm -v
9.6.2
```
ZED -> `0.77.3`

macOS: `12.6.3`

**Comment by bminde** [comment]
Zed uses the [Pyright](https://github.com/microsoft/pyright) language server for Python autocompletion. It works for me, but there are some reports of the automatic installation of Pyright failing. Se #1131 and #1020. You could check the log to see if that's the problem. Open the command palette (cmd + shift + p) and search for `zed: open log`. Some have got it working after upgrading Node to a newer version.

**Reply by JosephTLyons** [reply]
For what it's worth, we are currently working to resolve multiple issues with language servers failing to download.

**Reply by VedAustin** [reply]
Yep .. I noticed that it was complaining of Python language server not downloading.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
